python/resdata/util/test/resdata_test_runner.py

```diff
@@ -19,7 +16,7 @@ def findTestsInDirectory(path, recursive=True, pattern="test*.py"):
         loader = TestLoader()
         test_suite = loader.discover(path, pattern=pattern)
 
-        for root, dirnames, filenames in os.walk(path):
+        for root, dirnames, _ in os.walk(path):
```

Comment:
This entire file is never called in the code, but I am not that worried about testing it. Instead I think we should create an issue to deprecate it.